tools/profileBuilder/cmd/list.go

```diff
@@ -81,13 +84,43 @@ $> ../model/testdata/smallProfile.txt > profileBuilder list --name small_profile
 			errLog.Fatalf("failed to unmarshal JSON: %v", err)
 		}
 
+		if modulesFlag {
+			modver, err := getLatestModVer(outputRootDir)
```

Comment:
>getLatestModVer [](start = 18, length = 15)

this is the profile current mod ver ?

Reply:
Correct (will add comment for `getLatestModVer()`).